src/IB/CIBSaddlePointSolver.cpp

```diff
@@ -1102,8 +1102,8 @@ CIBSaddlePointSolver::monitorKSP(KSP ksp, int it, PetscReal rnorm, void* /*mctx*
     }
 
     std::streamsize old_precision = tbox::plog.precision(16);
-    tbox::plog << std::scientific << it << " SP_KSP " << print_normtype << " resid norm " << (double)rnorm
-               << " true resid norm " << (double)truenorm << " ||r(i)||/||b|| " << (double)(truenorm / bnorm)
+    tbox::plog << std::scientific << it << " SP_KSP " << print_normtype << " resid norm " << static_cast<double>(rnorm)
+               << " true resid norm " << static_cast<double>(truenorm) << " ||r(i)||/||b|| " << static_cast<double>(truenorm / bnorm)
```

Comment:
I don't think there is any reason to cast these: `PetscReal` is either `float` or `double` anyway.